src/server/pfs/fuse/server.go

```diff
@@ -944,17 +947,18 @@ func Server(sopts *ServerOptions, existingClient *client.APIClient) error {
 		w.Write(marshalled) //nolint:errcheck
 	})
 
-	// TODO: switch http server for gRPC server and bind to a unix socket not a
-	// TCP port (just for convenient manual testing with curl for now...)
-	// TODO: make port and bind ip parameterizable
-	srv := &http.Server{Addr: ":9002", Handler: router}
+	// Using unix domain socket
+	sockPath := sopts.SockDir + "/pfs_fuse_server.sock"
```

Comment:
This isn't conditional as it should be more like:
```
if sopts.SockPath:
   srv := &http.Server{Handler: router}
else: 
   srv := &http.Server{Addr: ":9002", Handler: router}
```

Reply:
ok